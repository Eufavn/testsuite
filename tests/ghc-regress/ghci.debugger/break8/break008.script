-- Check breakpointCond

:l A
f 1
f 4
:p i
:q