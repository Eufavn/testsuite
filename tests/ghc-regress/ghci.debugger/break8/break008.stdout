()
Stopped at a breakpoint in A.hs:3. Local bindings in scope:
  i :: a
A.hs:3:6-19> i = 4
A.hs:3:6-19> Returning to normal execution...
()
